src/css/styles.scss

```diff
@@ -0,0 +1,9 @@
+body {
+  background-color: #F9F9F9;
+}
+
+h1 {
+  font-family: monospace;
+  color: #1C4C55;
+  text-align: center;
+}
```

Comment:
What is the purpose of these styles?

Reply:
Just adding a little flavor to the example build. 😆 Whether or not the example build remains is still in question. See https://github.com/sparkbox/carbon/pull/25#discussion_r320339596